### PR TITLE
Add Other Programming Languages page

### PR DESCRIPTION
**Pull request by microbit-mark**
http://review.review.tech.microbit.org.s3-website-eu-west-1.amazonaws.com/software/other_languages/

(cherry picked from commit 663d9917d47ee1cc46de066b06f18bd1840b8dbc)
as per https://github.com/microbit-foundation/tech_microbit_org/pull/50#issuecomment-460344008

### CODE REVIEW

File: docs/software/other-langagues.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   loop
+      MicroBit.Display.Display ("<3");
+   end loop;
+end Main;
+```
+
+
+# Contributions
+
+This page can be edited in our [Github Repository](https://github.com/microbit-foundation/dev-docs/tree/master/software), with submission guidelines [found in our community issue tracker](https://github.com/microbit-foundation/microbit-community/issues/17).
```
Comment (microbit-mark) [inline docs/software/other-langagues.md]: The community issue tracker is not publically accessible yet, so we should probably remove the link to it for now.

Comment (jaustin) [reply docs/software/other-langagues.md]: Let's just merge the contents into tech.microbit.org for now? We can then discuss on the PR here.

Comment (microbit-mark) [reply docs/software/other-langagues.md]: Preview link in PR above

**Comment by microbit-sam** [comment]
Cool, thanks for moving it to dev-docs @microbit-mark!

**Pull request merged**